helm/cluster-vsphere/values.schema.json

```diff
@@ -132,6 +147,109 @@
+                                                        "name": "10-set-hostname.conf"
+                                                    }
+                                                ],
+                                                "enabled": true,
+                                                "name": "set-hostname.service"
+                                            },
+                                            {
+                                                "contents": {
+                                                    "install": {
+                                                        "wantedBy": [
+                                                            "default.target"
+                                                        ]
+                                                    },
+                                                    "unit": {
+                                                        "description": "Disable TCP segmentation offloading"
+                                                    }
+                                                },
+                                                "dropins": [
+                                                    {
+                                                        "contents": "[Unit]\nAfter=network.target\n[Service]\nType=oneshot\nRemainAfterExit=yes\nExecStart=/usr/sbin/ethtool -K ens192 tx-udp_tnl-csum-segmentation off\nExecStart=/usr/sbin/ethtool -K ens192 tx-udp_tnl-segmentation off",
+                                                        "name": "10-ethtool-segmentation.conf"
+                                                    }
+                                                ],
+                                                "enabled": true,
+                                                "name": "ethtool-segmentation.service"
+                                            },
+                                            {
+                                                "dropins": [
+                                                    {
+                                                        "contents": "[Unit]\n# kubeadm must run after coreos-metadata populated /run/metadata directory.\nRequires=coreos-metadata.service\nAfter=coreos-metadata.service\n[Service]\n# Make metadata environment variables available for pre-kubeadm commands.\nEnvironmentFile=/run/metadata/*",
+                                                        "name": "10-flatcar.conf"
+                                                    }
+                                                ],
+                                                "enabled": true,
+                                                "name": "kubeadm.service"
+                                            }
```

Comment:
this section duplicates [provider-specific](https://github.com/giantswarm/cluster-vsphere/blob/main/helm/cluster-vsphere/templates/_helpers.tpl#L278-L329) systemd units. currently, the `cluster` chart doesn't have the ability to template these units out fully, so a very basic unit file is created with the options available in the schema and then any other service args are added via a drop-in file. Currently these drop-ins have to be stringified as JSON doesn't support multi-line strings.

Reply:
github shows this as outdated as the last dropin for the kubeadm service isn't required so i've removed it - the `cluster` chart handles this now. The rest of this section is still valid.